### PR TITLE
media-libs/nvidia-vaapi-driver: new package, add 0.0.10

### PR DESCRIPTION
**Pull request by winterheart**
A VA-API implemention using NVIDIA's NVDEC.

### CODE REVIEW

**Comment by PF4Public** [comment]
It is worth mentioning somewhere in ebuild that it works _only_ with firefox.

File: media-libs/nvidia-vaapi-driver/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<pkgmetadata>
+	<maintainer type="person">
+		<email>azamat.hackimov@gmail.com</email>
+		<name>Azamat H. Hackimov</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<upstream>
```
Comment (juippis) [inline media-libs/nvidia-vaapi-driver/metadata.xml:12]: Please add the mozilla project here as proxy too.

**Review by juippis — APPROVED** [review]
Upstream says the file can be symlinked under **/usr/lib64/dri/vdpau_drv_video.so**, should we do that, or is it automagic if vdpau is detected build-time?

**Comment by juippis** [comment]
Also @ionenwks any comments?

**Comment by winterheart** [comment]
> Upstream says the file can be symlinked under **/usr/lib64/dri/vdpau_drv_video.so**, should we do that, or is it automagic if vdpau is detected build-time?

I don't know how VDPAU would be involved in. Most distributions don't make symlink. For info:

```
# vainfo 
Trying display: wayland
Trying display: x11
libva info: VA-API version 1.18.0
libva info: Trying to open /usr/lib64/va/drivers/nvidia_drv_video.so
libva info: Found init function __vaDriverInit_1_0
libva info: va_openDriver() returns 0
vainfo: VA-API version: 1.18 (libva 2.18.2)
vainfo: Driver version: VA-API NVDEC driver [egl backend]
vainfo: Supported profile and entrypoints
      VAProfileMPEG2Simple            : VAEntrypointVLD
      VAProfileMPEG2Main              : VAEntrypointVLD
      VAProfileVC1Simple              : VAEntrypointVLD
      VAProfileVC1Main                : VAEntrypointVLD
      VAProfileVC1Advanced            : VAEntrypointVLD
      VAProfileH264Main               : VAEntrypointVLD
      VAProfileH264High               : VAEntrypointVLD
      VAProfileH264ConstrainedBaseline: VAEntrypointVLD
      VAProfileHEVCMain               : VAEntrypointVLD
      VAProfileVP8Version0_3          : VAEntrypointVLD
      VAProfileVP9Profile0            : VAEntrypointVLD
      VAProfileAV1Profile0            : VAEntrypointVLD
      VAProfileHEVCMain10             : VAEntrypointVLD
      VAProfileHEVCMain12             : VAEntrypointVLD
      VAProfileVP9Profile2            : VAEntrypointVLD
```

**Review by juippis — APPROVED** [review]
I'll include some sort of a message on next firefox bump about this. Thanks!

**Pull request closed without merging**